src/System.IO.Pipes/tests/System.IO.Pipes.Tests.csproj

```diff
@@ -16,6 +16,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IncludePerformanceTests>true</IncludePerformanceTests>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetsWindows)' == 'true'">
+    <ProjectJson>win/project.json</ProjectJson>
+    <ProjectLockJson>win/project.lock.json</ProjectLockJson>
+  </PropertyGroup>
```

Comment:
We don't need to do that.  We follow this same pattern elsewhere.  Also, I hope to delete this after I put in the not supported assembly for Overlapped.